Gemfile.lock

```diff
@@ -279,13 +279,16 @@ GEM
       rubocop-ast (>= 1.28.0, < 2.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 2.4.0, < 3.0)
-    rubocop-ast (1.28.0)
+    rubocop-ast (1.28.1)
       parser (>= 3.2.1.0)
     rubocop-capybara (2.18.0)
       rubocop (~> 1.41)
-    rubocop-rspec (2.20.0)
+    rubocop-factory_bot (2.22.0)
```

Comment:
I was wondering where this came from:

https://github.com/rubocop/rubocop-rspec/blob/master/CHANGELOG.md#2220-2023-05-06

* Extract factory_bot cops to a separate repository, [rubocop-factory_bot](https://github.com/rubocop/rubocop-factory_bot). The rubocop-factory_bot repository is a dependency of rubocop-rspec and the factory_bot cops are aliased (RSpec/FactoryBot/Foo == FactoryBot/Foo) until v3.0 is released, so the change will be invisible to users until then. ([@ydah](https://github.com/ydah))